api/src/main/proto/spine/protodata/events.proto

```diff
@@ -38,252 +38,330 @@ option java_multiple_files = true;
 import "spine/protodata/ast.proto";
 import "spine/protodata/source.proto";
 
+option (every_is).java_type = "io.spine.protodata.FileAware";
+
 // Emitted when processing of a file begins.
 message FileEntered {
 
-    FilePath path = 1;
+    // The file processing of which is started.
+    File file = 1;
 
-    File file = 2;
+    // The header of the file.
```

Comment:
Let's append this notion of the equality of `file` and `header.file`, we do in another file in this changeset.